clients/src/test/java/org/apache/kafka/common/record/FileRecordsTest.java

```diff
@@ -316,7 +316,7 @@ public void testConvertNonCompressedToMagic1() throws IOException {
         try (FileRecords fileRecords = FileRecords.open(tempFile())) {
             fileRecords.append(records);
             fileRecords.flush();
-            Records convertedRecords = fileRecords.toMessageFormat(Record.MAGIC_VALUE_V1);
+            Records convertedRecords = fileRecords.toMessageFormat(Record.MAGIC_VALUE_V1, TimestampType.CREATE_TIME);
```

Comment:
Is there a reason for picking `CREATE_TIME` instead of `LOG_APPEND_TIME`? I guess either is wrong since the timestamp field will be `-1` in this case, right?